libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -2,36 +2,74 @@
 
 #include "unity.h"
 
-#include "mock_mqtt_lightweight.h"
-
 /* Include paths for public enums, structures, and macros. */
 #include "mqtt.h"
 
+/**
+ * @brief A valid starting packet ID per MQTT spec. Start from 1.
+ */
+#define MQTT_NEXT_PACKET_ID_START    ( 1 )
+
 /* ============================   UNITY FIXTURES ============================ */
+
+/* Called before each test method. */
 void setUp( void )
 {
 }
 
-/* called before each testcase */
+/* Called after each test method. */
 void tearDown( void )
 {
 }
 
-/* called at the beginning of the whole suite */
+/* Called at the beginning of the whole suite. */
 void suiteSetUp()
 {
 }
 
-/* called at the end of the whole suite */
+/* Called at the end of the whole suite. */
 int suiteTearDown( int numFailures )
 {
+    return numFailures;
```

Comment:
I don't see this logic referenced elsewhere in your change. Is it used by the framework?

Reply:
I am just using it to pass the number of failed tests to main. Do you suggest removing it?